### PR TITLE
Setup framework for translation.

### PR DESCRIPTION
**Pull request by bensonnw**
#315

### CODE REVIEW

**Comment by bensonnw** [comment]
@kathrynkodama , @turkeylurkey, thx

File: jakarta.eclipse/org.eclipse.lsp4jakarta.lsp4e.core/src/org/jakarta/lsp4e/JakartaLSConnection.java [hunk 1]
```diff
@@ -43,11 +44,21 @@
 public class JakartaLSConnection extends ProcessStreamConnectionProvider {
 
     public JakartaLSConnection() {
+
+//        Locale.setDefault(new Locale("my", "TEST")); // testing code
```
Comment (kathrynkodama) [inline jakarta.eclipse/org.eclipse.lsp4jakarta.lsp4e.core/src/org/jakarta/lsp4e/JakartaLSConnection.java:48]: Should we remove this?

File: jakarta.ls/src/main/java/org/eclipse/lsp4jakarta/commons/SnippetRegistry.java [hunk 2]
```diff
@@ -172,7 +173,8 @@ public List<CompletionItem> getCompletionItem(final Range replaceRange, final St
             String label = snippet.getPrefixes().get(0);
             CompletionItem item = new CompletionItem();
             item.setLabel(label);
-            item.setDetail(snippet.getDescription());
+//            item.setDetail(snippet.getDescription());
```
Comment (kathrynkodama) [inline jakarta.ls/src/main/java/org/eclipse/lsp4jakarta/commons/SnippetRegistry.java:176]: Looks like we can safely remove this line as well

File: jakarta.ls/src/main/resources/org/eclipse/lsp4jakarta/messages/messages.properties [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+# * terms of the Eclipse Public License v. 2.0 which is available at
+# * http://www.eclipse.org/legal/epl-2.0.
+# *
+# * SPDX-License-Identifier: EPL-2.0
+# *******************************************************************************/
+
+# NLS_MESSAGEFORMAT_VAR
+# NLS_ENCODING=UNICODE
+
+InitializingJakartaLS = Initializing Jakarta EE server.
```
Comment (kathrynkodama) [inline jakarta.ls/src/main/resources/org/eclipse/lsp4jakarta/messages/messages.properties:14]: Is this property in use?

File: jakarta.ls/src/main/java/org/eclipse/lsp4jakarta/commons/SnippetRegistry.java (hunk 2 above)
Comment (bensonnw) [reply jakarta.ls/src/main/java/org/eclipse/lsp4jakarta/commons/SnippetRegistry.java:176]: Keep the line there in order to have some other thoughts for JSON files.

File: jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/resources/org/eclipse/lsp4jakarta/jdt/core/messages/messages.properties [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+# * http://www.eclipse.org/legal/epl-2.0.
+# *
+# * SPDX-License-Identifier: EPL-2.0
+# *******************************************************************************/
+
+# NLS_MESSAGEFORMAT_VAR
+# NLS_ENCODING=UNICODE
+
+# Diagnostic messages
+AnnotationMustDefineAttribute = The annotation {0} must define the attribute ''{1}''.
```
Comment (turkeylurkey) [inline jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/resources/org/eclipse/lsp4jakarta/jdt/core/messages/messages.properties:15]: Is it possible to check the file in without the control-Ms? You may need to adjust your git configuration.

Comment (turkeylurkey) [reply jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/resources/org/eclipse/lsp4jakarta/jdt/core/messages/messages.properties:15]: Same issue in jakarta.ls.

Comment (bensonnw) [reply jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/resources/org/eclipse/lsp4jakarta/jdt/core/messages/messages.properties:15]: Not sure if we should do it using git configuration, as git could corrupt a lot of other files when git incorrectly "guesses" the line-endings or character is there but NOT as a line-ending.

Comment (turkeylurkey) [reply jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/resources/org/eclipse/lsp4jakarta/jdt/core/messages/messages.properties:15]: We have issue #321 to track this issue.

**Review by turkeylurkey — APPROVED** [review]

**Review by kathrynkodama — APPROVED** [review]

**Pull request merged**